### PR TITLE
Update GalaxyNG S3 URL

### PR DESCRIPTION
**Pull request by michellemacrh**
Update GalaxyNG S3 URL to reflect the new URL

[DDF] With the recent release of GalaxyNG, the new S3 URL is: https://ansible-galaxy-ng.s3.dualstack.us-east-1.amazonaws.com

https://issues.redhat.com/browse/AAP-16874

### CODE REVIEW

**Review by EMcWhinn — COMMENTED** [review]
Minor suggestions.

File: downstream/archive/archived-modules/platform/ref-network-ports-protocols.adoc [hunk 1]
```diff
@@ -5,12 +5,12 @@
 
 [role="_abstract"]
 
-{PlatformName} (AAP) uses a number of ports to communicate with its services. These ports must be open and available for incoming connection to the {PlatformName} server in order for it to work. Ensure that these ports are available and are not being blocked by the server firewall.
+{PlatformName} (AAP) uses several ports to communicate with its services. These ports must be open and available for incoming connection to the {PlatformName} server in order for it to work. Ensure that these ports are available and are not being blocked by the server firewall.
```
[the remark this answers is not in the document]
Comment (michellemacrh) [reply downstream/archive/archived-modules/platform/ref-network-ports-protocols.adoc:8]: Definitely, that reads better 👍  I'll correct that

File: downstream/archive/archived-modules/platform/ref-network-ports-protocols.adoc [hunk 2]
```diff
@@ -5,12 +5,12 @@
...
 
-The following tables provide the default {PlatformName} destination ports required for each application.
+The following tables show the default {PlatformName} destination ports required for each application.
 
 [NOTE]
-The default destination ports and installer inventory listed below are configurable. If you choose to configure them to suit your environment, you may experience a change in behavior.
+The default destination ports and installer inventory listed below are configurable. If you choose to configure them to suit your environment, you might experience a change in behavior.
```
[the remark this answers is not in the document]
Comment (michellemacrh) [reply downstream/archive/archived-modules/platform/ref-network-ports-protocols.adoc:13]: Thanks for highlighting this one, I need to get better at rewording these types of sentences 👍

**Review by EMcWhinn — APPROVED** [review]
LGTM

**Pull request merged**